### PR TITLE
Add FindBin to ecosystem

### PR DESCRIPTION
**Pull request by lembark**
see https://gitlab.com/lembark/Perl6-FindBin

- [X] I **agree** to the usage of the META file as listed [here](https://github.com/perl6/ecosystem#legal).

- [X] I have a license field listed in my META file that is one of https://spdx.org/licenses
  - [ ] My license is not one of those found on spdx.org but I **do** have a license field.
        In this case make sure you have a license URL listed under support. [See this example](https://github.com/samcv/URL-Find/blob/master/META6.json).
   - [ ] I **don't** have a license field. Yes, I understand this is **not recommended**.

### CODE REVIEW

File: META.list [hunk 1]
```diff
@@ -853,3 +853,4 @@ https://raw.githubusercontent.com/tony-o/perl6-libcsv/master/META6.json
 https://raw.githubusercontent.com/alabamenhu/Fluent/master/META6.json
 https://raw.githubusercontent.com/bduggan/p6-Grammar-PrettyErrors/master/META6.json
 https://raw.githubusercontent.com/ALANVF/P6TK/master/META6.json
+https://gitlab.com/lembark/Perl6-FindBin/blob/master/META6.json
```
Comment (JJ) [inline META.list]: You need to put the path to the raw JSON file.

**Comment by JJ** [comment]
There's an error with the source URL, even fixed. You need to do that yourself.

File: META.list (hunk 1 above)
Comment (lembark) [reply META.list]: Sanity check: The URL references the META6.json file?

wget and firefox both snag the JSON for me.

Q: What is wrong with the URL?
Q: Other than an https-link to the META6.json file, what do I need to include?

Comment (AlexDaniel) [reply META.list]: I think it needs to be this link: https://gitlab.com/lembark/Perl6-FindBin/raw/master/META6.json

Comment (timo) [reply META.list]: @lembark excuse my butting in here randomly, but I've sent you two mails from one account and another from a different account and I'd like to hear whether they are arriving or getting lost. thanks!

Comment (AlexDaniel) [reply META.list]: Also, I see `CPAN:lembark`. If you upload your module to CPAN, then nothing needs to be changed in this repository. http://modules.perl6.org/ and zef will pick up your module automatically (although with some delay, up to 1 day I think).

**Comment by momozor** [comment]
@lembark Consider changing the `git@gitlab.com:lembark/Perl6-FindBin.git' `  to `git://gitlab.com/lembark/Perl6-FindBin.git` or similar for the `source-url` property.

File: META.list (hunk 1 above)
Comment (momozor) [reply META.list]: @AlexDaniel I have a question. Will http://modules.perl6.org and zef index this module if the META6.json is wrongly formatted or if any of the property in the file doesn't work? (like incorrect URL for the source-url property)?

**Comment by JJ** [comment]
It will not pass the test, it will not be indexed.

**Comment by jonathanstowe** [comment]
For reference the ```check-sources``` of  ```Test::META``` would fail for ```git@gitlab.com:lembark/Perl6-FindBin.git``` as it is not a valid URI.

**Comment by JJ** [comment]
I have seen several new versions of this on CPAN. Should I close the PR?

**Comment by lembark** [comment]
At this point it should be working, I'd suggest grabbing whatever's most recent on CPAN.
I won't be updating this much going forward -- it seems to work at this point.

**Comment by lembark** [comment]
Q: What is this telling me about the '::'?
I don't find any double-colons in the json file...

Resolving deltas: 100% (121/121), done.

        ok 7 - name has a '::' rather than a hyphen (if this is intentional please pass :relaxed-name to meta-ok)
    ok 2 - Project META file is good

**Comment by Altai-man** [comment]
>Q: What is this telling me about the '::'?

The thing is, in Perl 6, distribution naming style "Foo::Bar" is preferred over "Foo-Bar", so this test checks absence of hyphens. Since your distribution name is single-piece(just "Foo", not "Foo::Bar"), it is naturally doesn't have any hyphens.

**Comment by lembark** [comment]
OK... should I add a trailing hyphen to the name or is it reasonable to have the test check for a singly-named moule?

**Comment by lembark** [comment]
Sanity check: If the module is added to CPAN as "FindBin" will it be available for use? Or is zef only capable of acquiring files that are in the ecosystem file?

**Comment by Altai-man** [comment]
>OK... should I add a trailing hyphen to the name or is it reasonable to have the test check for a singly-named moule?

You don't need to do anything, the test is `ok`ed, if it is not, then it is definitely a bug. But your "ok 2 - Project META file is good" means META file is good enough.

>Sanity check: If the module is added to CPAN as "FindBin" will it be available for use? Or is zef only capable of acquiring files that are in the ecosystem file?

Zef works with both CPAN and ecosystem file.

**Pull request merged**